src/main/java/pl/lukakan/todoapp/model/TaskCategory.java

```diff
@@ -0,0 +1,5 @@
+package pl.lukakan.todoapp.model;
+
+public enum TaskCategory {
+    COURSE, HOMEWORK, WORK, TRAINING, SHOPPING;
```

Comment:
Z enumami do kategorii raczej trzeba ostrożnie podchodzić. W przypadku jakbyśmy często chcieli dodawać kategorię, to lepiej nie tworzyć enuma. Tutaj raczej jest ok.

Reply:
w sumie wrzuciłem enumy, bo tak zrozumiałem to, że zadania muszą mieć kategorie. Jak wrzucimy normalnego Stringa definiowanego przez uzytkownika, to bedzie potem problem w przypadku np filtracji po kategoriach. Chyba zeby dolozyc tabele z kategoriami i opcje ich dodawania przez usera?